### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by calebporzio** [alpinejs/alpine#481] [4. General]
Hey Folks,

Welcome to Alpine's "Discussions" section. Feel free to introduce yourself!

My goal is that this humble tab on the repository becomes a robust resource of information and discussion.

Have a feature request? Let's hear it!

Have something to show off? Let's see it!

Let it fly 🥏 👊

**Comment by ryangjchandler** [comment]
Hello everyone,

Ryan here! Looking forward to hearing everyone's thoughts and ideas for Alpine. Got any questions or feature requests, let us know!

**Reply by calebporzio** [reply]
👋 Ryan!

**Comment by calebporzio** [comment]
[This is a "null" reply so I can mark this thread as "Answered" 👀]

**Reply by natedunn** [reply]
Hmm, yes, I concur

**Comment by mikeziri** [comment]
hi all! wow, 7.5k stars already. keep up the good work. missing project-x 😄

**Reply by ryangjchandler** [reply]
Hey Mike, thanks for stopping by!

**Comment by lvltech-mx** [comment]
It would be great to add a x-validate input to check if the input is blank or not.

**Reply by lokmanm** [reply]
Kinda similar questions, what are the plans on adding validation logic? If there are any ofc.

**Reply by ryangjchandler** [reply]
Would be great to move these ideas into a separate thread so they don't get lost in the conversation! :)

**Comment by paulcsmith** [comment]
Hi Caleb! 

Huge fan of your work and Alpine.js. I'm on the discussions team at GitHub so if you have any feedback feel free to reach out! I'm happy to relay it to the team

Can't wait to see Alpine grow even bigger and see what y'all do with Discussions :D

**Comment by wobsoriano** [comment]
Hello! I'm using alpine on a browser extension currently. Pretty easy to use 👍

Would love to see a sample on how to implement computed properties. Something what like Vue does.

Thanks for this!
